test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/EndToEndTestFixture.cs

```diff
@@ -82,17 +82,15 @@ public async Task InitializeAsync()
             }
 
             string logPath = Path.Combine(Path.GetTempPath(), @"Functions");
+            if (!string.IsNullOrEmpty(_functionsWorkerRuntime))
+            {
+                Environment.SetEnvironmentVariable(LanguageWorkerConstants.FunctionWorkerRuntimeSettingName, _functionsWorkerRuntime);
+            }
```

Comment:
Just catching up on this -- Is there a reason this has to be an env variable and we can't use the IConfguration abstraction below?

Reply:
Oh, actually, n/m. We should be using `TestEnvironment` to make it more reliable (trying to avoid setting global env vars whenever possible). But I see a bunch of tests using real env vars so it'll be tough to swap out (and likely not worth it).

Reply:
Hopefully the ones you see are legacy tests that haven't been migrated yet. We're trying to switch over as we go.

Reply:
Yeah, we should switch them all. There's a bunch here, for example: https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs#L397. Those started failing when I switched this. We can tackle that in a separate PR -- it was growing a bit to big to be done in the one I'm putting together.